docs/LV_client/tutorials/wheel_integration.md

```diff
@@ -0,0 +1,31 @@
+# Wheel integration
+
+## Pre-requisites
+1. The **mono_closed_loop_example_wheel.vi** and the **mono_replay_example_wheel.vi** use the **G920 for Xbox One and PC**, more information [here](https://www.logitechg.com/en-us/products/driving/driving-force-racing-wheel.html).
```

Comment:
Are there any other .vi programs the user can use or create on their own? If not, making this explicit in the introduction could really clean up this section such as: 1.  **G920 for Xbox One and PC**, more information [here](https://www.logitechg.com/en-us/products/driving/driving-force-racing-wheel.html).